inspirehep/dojson/hep/fields/bd70x75x.py

```diff
@@ -66,19 +62,64 @@ def _get_affiliations(value):
                     record = None
 
                 result.append({
-                    'curated_relation': record is not None,
                     'record': record,
                     'value': value,
                 })
         else:
             for value in institutions:
                 result.append({
-                    'curated_relation': False,
                     'value': value,
                 })
 
         return result
 
+    def _get_ids(value):
+        def _is_jacow(j_value):
+            return j_value.upper().startswith('JACOW-')
+
+        def _is_orcid(j_value):
+            return j_value.upper().startswith('ORCID:') and len(j_value) > 6
+
+        def _is_naked_orcid(j_value):
```

Comment:
why not [idutils.is_orcid](http://idutils.readthedocs.io/en/latest/#idutils.is_orcid)?